server/service.go

```diff
@@ -646,6 +651,143 @@ func (m *BoostService) processCapellaPayload(w http.ResponseWriter, req *http.Re
 	m.respondOK(w, result)
 }
 
+func (m *BoostService) processDenebPayload(w http.ResponseWriter, req *http.Request, log *logrus.Entry, blindedBlock *eth2ApiV1Deneb.SignedBlindedBeaconBlock) {
+	// Get the slotUID for this slot
+	slotUID := ""
```

Comment:
I think we should rename this variable so it doesn't shadow the type:

https://github.com/flashbots/mev-boost/blob/c91b043e382fb1e7c0ac496b10ed0f715ea99737/server/service.go#L60-L63